lib/src/components/json_widget_registry.dart

```diff
@@ -59,6 +61,8 @@ class JsonWidgetRegistry {
         _parent = parent {
     _logger = Logger('REGISTRY ${this.debugLabel}');
     _customBuilders.addAll(builders ?? {});
+    _internalFunctions =
+        internalFunctions ?? JsonWidgetInternalFunctions.defaults();
```

Comment:
Why allow "internalFunctions" and "functions" to both be passed in?

Reply:
I think that we have to keep it separately - to not break existing apps
If we would have only `functions` then passing any custom function will remove all internal ones?

I'm open to suggestions. I've kept it separately for the backward compatibility

Reply:
I'd do something like:
```
_internalFunctions = {
  ...JsonWidgetInternalFunctions.defaults(),
  ...(functions ?? <String, JsonWidgetFunction>{})
}
```

And get rid of `_functions`.